src/org/usfirst/frc/team467/robot/DriverStation.java

```diff
@@ -102,5 +106,29 @@ public double getArcadeSpeed() {
 	public double getArcadeTurn() {
 		return getDriveJoystick().getRightStickX();
 	}
+	
+	public double limitAccelaration(double newSpeed) {
+		if (newSpeed > previousSpeed + maxAcceleration) {
+			previousSpeed += maxAcceleration;
+		}
+		
+		else if(newSpeed < previousSpeed - maxAcceleration) {
+			previousSpeed -= maxAcceleration;
+		}
+		
+		return previousSpeed;
+	}
```

Comment:
I don't think this is right. What happens if the previous speed is 0.2, maxAcceleration is 0.1, and the new speed is 0.28? There's a similar case for the negative.